### PR TITLE
Azezetl bug fixes

### PR DESCRIPTION
**Pull request by encodis**

### CODE REVIEW

File: azezetl/src/azezetl/steps/validators.py
Comment (WilliamGasson) [inline azezetl/src/azezetl/steps/validators.py:1]: Are you going to move some of this to azqccheck?

**Review by WilliamGasson — COMMENTED** [review]
Looks good to me, mostly seems to be updating the get so it takes step type and creating azqccheck

File: azezetl/tests/__init__.py
[the remark this answers is not in the document]
Comment (encodis) [reply azezetl/tests/__init__.py:1]: Yeah there's a lot of tidying needed!

File: azezetl/tests/test_pipeline.py [hunk 1]
```diff
@@ -70,14 +70,17 @@ def test_Pipeline_combo(self):
                 column_map: {'C': 'D'}
         """
 
+        ## fails because with data split, you are looking for DropColumns_1 in
+        ## validation storage, which does not exist
```
[the remark this answers is not in the document]
Comment (encodis) [reply azezetl/tests/test_pipeline.py:74]: Not sure - I'll have to check. Could well be an old comment

File: azezetl/src/azezetl/steps/validators.py
Comment (encodis) [reply azezetl/src/azezetl/steps/validators.py:1]: Yes, ideally all the validation steps will call functions in azqccheck

**Pull request merged**